CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+### Updated
+
+* The default version of the widget is `3.x` now.
+* Added `integration` to default config to set info
+  about the CKeditor version and the plugin version
+  for User Agent with requests to Uploadcare Upload API.
```

Comment:
* The default widget version is now `3.x`.
* Added `integration` to the default config to provide info on the CKEditor version and plugin version to User Agent with requests to `[Uploadcare Upload API](https://uploadcare.com/docs/api_reference/upload/)`.